docs/self-managed/deployment/helm/cloud-providers/kind.md

```diff
@@ -0,0 +1,466 @@
+:::
+
+| Component            | URL                                       |
+| -------------------- | ----------------------------------------- |
+| Zeebe Gateway (gRPC) | localhost:26500                           |
+| Zeebe Gateway (HTTP) | http://localhost:8080/                    |
+| Operate              | http://localhost:8080/operate             |
+| Tasklist             | http://localhost:8080/tasklist            |
+| Identity             | http://localhost:8080/identity            |
+| Management Identity  | http://localhost:18081/managementidentity |
+| Optimize             | http://localhost:8083                     |
+| Web Modeler          | http://localhost:8070                     |
+| Console              | http://localhost:8087                     |
+| Connectors           | http://localhost:8085                     |
+| Keycloak             | http://localhost:18080                    |
+
+:::tip Connecting to the workflow engine
+To interact with the Camunda workflow engine via Zeebe Gateway using the [Orchestration Cluster REST API](/apis-tools/orchestration-cluster-api-rest/orchestration-cluster-api-rest-overview.md) or a local client/worker, connect to `localhost:26500` (gRPC) or `http://localhost:8080` (REST).
+:::
+
+At any time, run `kubectl get services -n camunda` to get a full list of deployed Camunda components and their network properties.
+
+</TabItem>
+
+</Tabs>
+
+### Default credentials
+
+- **Username**: `admin`
+- **Password**: Run the following script
+
+```bash reference
+https://github.com/camunda/camunda-deployment-references/blob/feature/kind-local/local/kubernetes/kind-single-region/procedure/get-password.sh
+```
+
+## Cleanup
+
+:::warning Destructive action
+This will destroy all data of Camunda 8 in the local development cluster.
+:::
+
+```bash
+# Uninstall Camunda
+helm uninstall camunda -n camunda
+
+# Delete cluster
+kind delete cluster --name camunda-platform-local
+
+# (domain mode) Remove hosts entries (requires sudo)
+sudo sed -i '/camunda.example.com/d' /etc/hosts
+
+# (domain mode) Clean certificates
+rm -rf .certs
+```
+
+## Troubleshooting
+
+### Pods don't start
+
+Check the pod status and events:
+
+```bash
+kubectl get pods -n camunda
+kubectl describe pod <pod-name> -n camunda
+kubectl logs <pod-name> -n camunda
+```
+
+### Browser shows certificate errors (Domain mode)
+
+Ensure the mkcert CA is installed:
+
+```bash
+mkcert -install
+```
+
+Regenerate certificates:
+
+```bash
+./procedure/certs-generate.sh
+./procedure/certs-create-secret.sh
+kubectl rollout restart deployment -n camunda
+```
+
+### Ingress doesn't work (Domain mode)
+
+Check the Ingress controller status:
+
+```bash
+kubectl get pods -n ingress-nginx
+kubectl get ingress -n camunda
+```
+
+### Insufficient resources
+
+Ensure Docker has enough resources allocated (4+ CPU cores, 8GB+ RAM). Check Docker Desktop settings or `/etc/docker/daemon.json`.
```

Comment:
For Docker Desktop settings could link to https://docs.docker.com/desktop/settings-and-maintenance/settings/#resources

I mean if I use Docker Desktop the file does not exist, maybe not as clear.

> cat: /etc/docker/daemon.json: No such file or directory

Maybe link to docker daemon - https://docs.docker.com/engine/daemon/
If you use the engine directly then depending on the OS it differs.